src/api/public/apidocs-new/paths/distributions_include_remotes.yaml

```diff
@@ -0,0 +1,74 @@
+get:
+  summary: List all distributions.
+  description: |
+    List all distributions that can be build against, including the one provided by the interconnect.
+  security:
+    - basic_authentication: []
+  responses:
+    '200':
+      description: |
+        OK. The request has succeeded.
+
+        XML Schema used for body validation: [distributions.rng](../schema/distributions.rng)
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/distributions.yaml'
+          example:
```

Comment:
@jcronenberg this is a pretty big example, since you have this 200 response two times (also in the `GET /distributions`), with the same example, I would move it to its own file in the responses folder and then reference it. A little less repeated code :)